vizro-core/examples/visual-vocabulary/README.md

```diff
@@ -70,7 +70,7 @@ The dashboard is still in development. Below is an overview of the chart types f
 | Correlation matrix    | ❌     | Correlation              |                                                                                                                                 |                                                                                                                                                                                                 |
 | Histogram             | ✅     | Distribution             | [Histograms with px](https://plotly.com/python/histograms/)                                                                     | [px.histogram](https://plotly.github.io/plotly.py-docs/generated/plotly.express.histogram)                                                                                                      |
 | Line                  | ✅     | Time                     | [Line plot with px](https://plotly.com/python/line-charts/)                                                                     | [px.line](https://plotly.com/python-api-reference/generated/plotly.express.line)                                                                                                                |
-| Lollipop              | ❌     | Ranking, Magnitude       |                                                                                                                                 |                                                                                                                                                                                                 |
+| Lollipop              | ✅     | Ranking, Magnitude       | [Lollipop & Dumbbell Charts with Plotly](https://towardsdatascience.com/lollipop-dumbbell-charts-with-plotly-696039d5f85)       | [px.scatter](https://plotly.com/python-api-reference/generated/plotly.express.scatter)                                                                                                          |
```

Comment:
Do you think we should reference Medium articles that are available only for the Medium subscribed users? I think users have to pay to read the entire article.

Reply:
Good question! 

In this case, yes - I think it's fine. This section is primarily to give credit to where the original code came from. For users, our optimized and refactored code snippet is the main one to use. So as long as they can access that, it's fine 👍 

If there are public articles out, that provide the same content, of course we can also prefer these, but in the lollipop example the public available ones did not use the code I used in the end. They use shapes, which I thought was a bit more difficult to understand. So the inspiration for this lollipop code is really coming from the medium article.